packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -73,14 +73,52 @@ class ExpansionTile extends StatefulWidget {
     this.collapsedShape,
     this.clipBehavior,
     this.controlAffinity,
-  }) : assert(initiallyExpanded != null),
+  }) : child = null,
+  assert(initiallyExpanded != null),
        assert(maintainState != null),
        assert(
        expandedCrossAxisAlignment != CrossAxisAlignment.baseline,
        'CrossAxisAlignment.baseline is not supported since the expanded children '
            'are aligned in a column, not a row. Try to use another constant.',
        );
 
+  /// a constructor for the [ExpansionTile] that allows a single [child] instead of a list of [children].
+  const ExpansionTile.single({
+    super.key,
+    this.leading,
+    required this.title,
+    this.subtitle,
+    this.onExpansionChanged,
+    required this.child,
+    this.trailing,
+    this.initiallyExpanded = false,
+    this.maintainState = false,
+    this.tilePadding,
+    this.expandedCrossAxisAlignment,
+    this.expandedAlignment,
+    this.childrenPadding,
+    this.backgroundColor,
+    this.collapsedBackgroundColor,
+    this.textColor,
+    this.collapsedTextColor,
+    this.iconColor,
+    this.collapsedIconColor,
+    this.shape,
+    this.collapsedShape,
+    this.clipBehavior,
+    this.controlAffinity,
+  }) : children = const <Widget>[],
+       assert(initiallyExpanded != null),
+       assert(maintainState != null),
+       assert(
+       expandedCrossAxisAlignment != CrossAxisAlignment.baseline,
+       'CrossAxisAlignment.baseline is not supported since the expanded children '
+           'are aligned in a column, not a row. Try to use another constant.',
+       );
```

Comment:
Indentation was wrong before your change.
Thank you to fix it:
- add one indent before `expandedCrossAxisAlignment`.
- add one indent before `'CrossAxisAlignment...`
- align `'are aligned in...` with previous line

Reply:
fixed indents